package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "commando-covid",
     "version": "0.1.0",
-    "homepage": "https://simurea.nancyclotep.com/modsir19test",
+    "homepage": ".",
```

Comment:
This config is saying "use wherever the index.html is as the relative reference for the routing"